app/helpers/application_helper.rb

```diff
@@ -8,4 +11,32 @@ def full_title(page_title = "")
       page_title + " | " + base_title
     end
   end
+
+  def email_slack_invite(user_email)
```

Comment:
This is the first time I see a helper used for something else than extracting logic from the views. We usually use service classes (there are already some examples in `app/services`).